civil.go

```diff
@@ -43,9 +45,11 @@ func DateOf(t time.Time) Date {
 	return d
 }
 
+const rfc3339Date = "2006-01-02"
```

Comment:
these consts are exported in stdlib time, it's probably useful to do it here for other users so they can use them for formatting strings etc.

Reply:
Good point! Fixed.